omnibus/files/private-chef-cookbooks/private-chef/recipes/config.rb

```diff
@@ -81,6 +81,32 @@
     PrivateChef.from_file(chef_server_path)
   end
 
+  # Add support for the external search case
+  if PrivateChef['opscode_solr4']['external']
+    # Disable internal elasticsearch
+    node.default['private_chef']['rabbitmq']['enable'] = false
+    node.default['private_chef']['opscode-solr4']['enable'] = false
+    node.default['private_chef']['opscode-expander']['enable'] = false
```

Comment:
Are we sure we want to disable these?  That seems inconsistent with the values for search_provider and search_queue_mode below.